api/lib/domain/models/Challenge.js

```diff
@@ -104,8 +107,8 @@ class Challenge {
   }
 
   // Same than isActive for algo
-  isPublished() {
-    return ['validé', 'validé sans test', 'pré-validé'].includes(this.status);
+  isPublished(useOnlyValidated) {
```

Comment:
En utilisant un objet, c'est plus facile de comprendre l'appel à la fonction.
```suggestion
  isPublished({ useValidatedOnly) ) {
```